### ISSUE TITLE
Overall status of Plesk + WHMCS project

### ISSUE THREAD

**Issue by brentv** [yola/p-plesk#19]
@akugaevskiy, I wanted to check in with you regarding the status of the [Plesk + WHMCS project](https://github.com/yola/p-plesk/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Plesk+WBB+Release+with+WHMCS%22)? 

It would be helpful to have a better understanding of where we stand with QA, what the remaining blockers are and what we should expect in terms of next steps after QA is complete?

**Comment by akugaevskiy** [comment]
The big blocker now is #2 (I consider #18 to be critical, not blocker).
 
After #2 is fixed, we can verify that the rest of the reported issues are addressed and we can perform full testing. If new issues are found, we'll report them. If no new issues are found, we should be good to go on the technical side (assuming we have video tutorials, documentation and onboarding form).

Hope this clears things up.

**Comment by robertbray** [comment]
Could you just do the first steps @akugaevskiy? so we could unblock #2 ?  I could invite you into be the Odin rep if you send me your email and mobile phone.

**Comment by akugaevskiy** [comment]
No, this should be done by our QA engineers to ensure proper end-to-end testing.


Andrey Kugaevskiy
Program Manager, Plesk
http://www.plesk.com<http://www.plesk.com/>
________________________________
